src/pages/index.tsx

```diff
@@ -13,7 +13,7 @@ import HomeLayout from 'layouts/Home';
 
 const IndexPage: React.FC = () => {
   return (
-    <RootLayout hasSkipNav={false}>
+    <RootLayout hasSkipNav={false} path="/">
```

Comment:
Alternatively, could use side effect to determine `location.pathname` and set some state. I decided to go with prop-drilling and avoid setting up state and side effects.